README.md

```diff
@@ -1,14 +1,91 @@
 
-## If you are an enthusiast who would like to bring smiles to people's lives we invite you to take part in this effort by contributing ❤️
+### Core Services
+•Expert Backed Medical Information: Patients can access verified, up to date information about various rheumatic conditions, curated by specialist rheumatologists.
+•Smart Appointment System: patients easily be able to use booking system for consultations with rheumatology specialists.
+•Medication Guide: Patients can get comprehensive information about treatment options, medication details, and possible side effects.
+•Multilingual Support: All information available in three languages (English, Sinhala, Tamil) to ensure accessibility for all the patients.
 
-You can contribute to the project in 2 ways,
-- Technically (coding, ui-design)
-- Non-technically (documentation, graphic design, etc)
+### Why RheumaConnect?
+Rheumatic patients often face several challenges,
+
+• Limited access to reliable medical information about their conditions.
+• Difficulty in understanding complex medical terminology.
+• Long waiting times for specialist consultations.
+• Confusion about medication and treatment options.
+• Language barriers in accessing medical information.
+
+RheumaConnect is transforming rheumatic care by offering an integrated approach to patient support. The platform provides free access to expert verified medical resources, with easy to understand explanations of conditions and treatments. Patients can quickly book appointments, reducing waiting times, and access clear medication guidelines along with detailed treatment information. With multi-language support, all resources are available in patients' preferred languages, ensuring better understanding and promoting improved healthcare outcomes.
+
+---
+
+## 🌟 Key Features
+
+• Tri-lingual support (English, Sinhala, Tamil).
+• Expert verified medical information.
+• Free medical advice for managing rheumatic conditions.
+• Appointment booking system with rheumatology specialists.
+• Treatment options and medication guides.
+• User-friendly interface designed for accessibility.
+• Comprehensive resource library for various rheumatic conditions.
+• Mobile responsive design for access anywhere, anytime.
+
+---
+
+## 🚀 Current State
+RheumaConnect has successfully completed the design phase and the project is now in the development phase. Currently, the team is focused on building the platform's core features.
+
+### Roadmap
+Phase 1: Website development and initial content population
+Phase 2: Integration of multilingual support
+Phase 3: Continuous content updates and feedback integration
+
+---
+
+## 👥 Project Team
+
+### Project Initiators
```

Comment:
Shall we drop this section for now? I think better to ask the prof about this and mention maybe.